pims/cine.py

```diff
@@ -520,7 +522,12 @@ def __len__(self):
 
     def get_time(self, i):
         '''Returm the time of frame i in seconds.'''
-        return float(i) / self.frame_rate
+        # TODO: This is not guaranteed to be the actual time.
```

Comment:
Guess we never used the external sync feature in the lab.

I have some recollection of Dustin reporting that if you changed the frame rate mid-collection the images came out garbled (I assume due to electronics not liking their clock rates changed quickly).